### ISSUE TITLE
SMS with > 160 characters does not send

### ISSUE THREAD

**Issue by rrsilaya** [globelabs/api#9]
I came across this situation that whenever I try to send an SMS with more than 160 characters, it simply does not send. As of now, my temporary fix is to chunk the SMS into 160-character messages.

Also, some information on receiving multi-part SMS such as `multPartRefId` are not included in the payload.

**Comment by adin234** [comment]
yes, that is the expected behavior as per the [old](https://github.com/globelabs/api/blob/d55b376ee872e54d85f75392d98aa01d561b0a64/Sms.md#sending) and [new](https://globelabs.github.io/docs/#sms-sending-sms-sms-mt) docs

**Comment by rrsilaya** [comment]
It says in the documentation that

> You can send multi-part sms **like you would in sending a normal sms**, just keep in mind that you will be charged per 160 characters.

**Comment by adin234** [comment]
are you using any wrapper class? if yes, will you mind telling us which wrapper class you used?

**Comment by rrsilaya** [comment]
No, I believe that I'm not using any wrapper classes. As I said, sending messages less than 160 characters sends but sending above that doesn't. As it was stated in the docs, the API can send messages more than 160 characters, that is a multi-part SMS, _like you would in sending a normal sms_. I don't see anything wrong with what I'm doing previously if that is the case.

**Comment by sinnedmi** [comment]
how can i send message po?
i'm always redirected to registration screen? maraming salamat po
https://developer.globelabs.com.ph/dialog/oauth?app_id=*********************

sorry for this question

**Comment by MikeLince** [comment]
tanong naman po.

i'm using .net. nakuha ko na po yung token at na authorize na rin, kaso kapag mag send nako sa SMS nag error siya "Unauthorized"

gamit kong API
GlobeLapAPI

var data = api1.PushSms(shortCode, payload); <- dito po siya nag error
